.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+ name: kube2py
+ namespace: mateapp
+ labels:
+   app: kube2py
+spec:
+ selector:
+   matchLabels:
+     app: kube2py
+ template:
+   metadata:
+     labels:
+       app: kube2py
+   spec:
+     containers:
+     - name: busybox
+       image: ikulyk404/busyboxplus:curl
```

Comment:
The DaemonSet is missing a command to execute a `curl` command every 5 seconds to a cluster IP service of the todo app. You need to add this command to ensure the DaemonSet performs its intended function.